app/views/steps/disbursement_type/edit.html.erb

```diff
@@ -0,0 +1,28 @@
+<% title t('.page_title') %>
+<% step_header(path: applications_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l"><%= t('.heading') %></h1>
+    <%= govuk_error_summary(@form_object) %>
+    <%= step_form @form_object do |f| %>
+      <%= f.govuk_date_field :disbursement_date, maxlength_enabled: true, legend: { size: 's' } %>
+
+      <%= f.govuk_radio_buttons_fieldset :disbursement_type, legend: { size: 's' } do %>
+        <% DisbursementTypes.values.each do |disbursement_type| %>
+          <%= f.govuk_radio_button :disbursement_type, disbursement_type.to_s do %>
+            <% if disbursement_type == DisbursementTypes::OTHER %>
+              <%= f.govuk_collection_select :other_type, OtherDisbursementTypes.values, :to_sym, :translated,
+                              data: { module: 'accessible-autocomplete' }, width: 20,
+                              options: { include_blank: true }, label: nil %>
+            <% end %>
+          <% end %>
+        <% end %>
+      <% end %>
+
```

Comment:
This may be me not understanding how the options get picked up from steps.yml - but how will the rate parameter get added into this? Is this logic that comes from the Engine?
![image](https://github.com/ministryofjustice/laa-claim-non-standard-magistrate-fee-backend/assets/26544538/81e1acf9-c4ba-44ec-8eeb-95dc728853d9)

Reply:
Fixed this and added rate to the pricing file so it can change easily in the future.